phoenicis-dist/src/resources/Phoenicis.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Name=Phoenicis PlayOnLinux
+GenericName=Game;Utility
```

Comment:
This should be in `Categories` and not here.

Reply:
Done